### PR TITLE
fix: chat view upside down

### LINKED ISSUE #79: Organize Chat View.
### Explanation:

The chat is upside down. The user chat should be located on the right side and the Delivery on the left side, check that it looks good in the different resolutions, I attach a Figma link if necessary for the view, el icono que aparece en la parte superior no es de el usuario ,es de el delivery, realizar ese cambio de imagen.

### Screenshots
Vista chat

### Checklist

- [ ] Change user chat and delivery site.
- [ ] Change the icon used in the header.

### References:

(https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/Dasher?node-id=420%3A332)

### PR DESCRIPTION
**Pull request by mdibuhossain**
fixes #79 

#### Screenshots
image

image

### Checklist:

- [x] Change user chat and delivery site.
- [x] Change the icon used in the header.

### CODE REVIEW

**Review by e-azocar — APPROVED** [review]
Good job

File: src/pages/Chat/Chat.styled.jsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const HeaderChatImg = styled.img`
   margin: auto 10px;
   width: 4vw;
   min-width: 40px;
-  max-width: 50px;
+  max-width: 50px;
```
Comment (Israel-Laguan) [inline src/pages/Chat/Chat.styled.jsx]: nit: remove extra space

File: src/pages/Chat/Chat.styled.jsx [hunk 2]
```diff
@@ -81,23 +78,33 @@ export const FooterChatInput = styled.form`
...
+    }
+  }
+
   div {
     display: flex;
-    justify-content: center;
+    justify-content: space-between;
     align-items: center;
-    margin: 0 auto;
+    margin: 0px auto;
```
Comment (Israel-Laguan) [inline src/pages/Chat/Chat.styled.jsx]: px is not needed when 0

Comment (mdibuhossain) [reply src/pages/Chat/Chat.styled.jsx]: sorry for the trouble. I fixed it.

**Review by Israel-Laguan — APPROVED** [review]

File: src/pages/Chat/Chat.styled.jsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import styled from '@emotion/styled'
 import { bg, p, rounded, size, text } from '../../styles/mixins'
 
 export const ChatLayoutContainer = styled.div`
-  ${[ size({ width: '100vw', minHeight: '100vh' }), bg.white ]}
+${[ size({ width: '100vw', minHeight: '100vh' }), bg.white ]}
```
Comment (heliomar-pena) [inline src/pages/Chat/Chat.styled.jsx]: check indentation

**Review by heliomar-pena — APPROVED** [review]
Good job! Please check indentation

File: src/pages/Chat/Chat.styled.jsx (hunk 3 above)
Comment (mdibuhossain) [reply src/pages/Chat/Chat.styled.jsx]: thanks, I hadn't noticed that.
It is fixed now.

**Pull request merged**